cpp/include/cudf/groupby.hpp

```diff
@@ -74,6 +74,12 @@ struct Options {
   bool const ignore_null_keys{true};
 };
 
+/**---------------------------------------------------------------------------*
+ * @brief Supported aggregation operations
+ *
+ *---------------------------------------------------------------------------**/
+enum operators { SUM, MIN, MAX, COUNT, MEAN };
```

Comment:
These operators should remain name spaced because there are operations that are unique to sort-based groupby that hash based groupby does not support.

Reply:
I got it